example/glue/configure_includes/configure_common.mk

```diff
@@ -24,5 +24,6 @@ CONFIGURE_ARGS += \
   --enable-OMR_OMRSIG \
   --enable-fvtest \
   --enable-OMR_GC_SEGREGATED_HEAP \
+  --enable-OMR_GC_MODRON_CONCURRENT_MARK \
```

Comment:
I would like to see evidence that new Glue APIs are properly if defined so that if a new language  not using that feature can compile.  Two of our recent features did not do this and you could not compile without those feature flags enabled